integration-test/875-camp-grounds-zoom.py

```diff
@@ -0,0 +1,49 @@
+# update landuse to include campground features and fix label zoom
+
+# large campground in landuse zoom 16
+# http://www.openstreetmap.org/way/237314510
+assert_has_feature(
+    16, 10599, 25679, 'landuse',
+    { 'kind': 'camp_site'})
+
+# large campground in point zoom 16
+# http://www.openstreetmap.org/way/237314510
+assert_has_feature(
+    16, 10599, 25679, 'pois',
+    { 'kind': 'camp_site'})
+
+# large campground in landuse zoom 13
+# http://www.openstreetmap.org/way/237314510
+assert_has_feature(
+    13, 1324, 3209, 'landuse',
+    { 'kind': 'camp_site'})
+
+# large campground in point zoom 13
+# http://www.openstreetmap.org/way/237314510
+assert_has_feature(
+    13, 1324, 3209, 'pois',
+    { 'kind': 'camp_site'})
+
+# small campground in landuse zoom 16
+# http://www.openstreetmap.org/way/417405356
+assert_has_feature(
+    16, 10471, 25326, 'landuse',
+    { 'kind': 'camp_site'})
+
+# small campground in point zoom 16
+# http://www.openstreetmap.org/way/417405356
+assert_has_feature(
+    16, 10471, 25326, 'pois',
+    { 'kind': 'camp_site'})
+
+# small campground in landuse zoom 13
+# http://www.openstreetmap.org/way/417405356
+assert_has_feature(
+    13, 1308, 3165, 'landuse',
+    { 'kind': 'camp_site'})
+
+# small campground in point zoom 13
+# http://www.openstreetmap.org/way/417405356
+assert_no_matching_feature(
+    13, 1308, 3165, 'pois',
+    { 'kind': 'camp_site'})
```

Comment:
Possibly we should be changing the tests? It seems like there's two different use-cases:

1. I want to test that _something_ here has some properties. E.g: highways drop bridge/tunnel at some zoom level. I don't care what the highways are: they should all have this behaviour.
2. I want to test that a specific feature is transformed in a specific way.

The former could be done with a tile coordinate and general matching, whereas the second needs the ID and possibly the zoom level - after we've downloaded the feature, we can figure out the tile coords. This would make us robust to movements of features across tile boundaries.

I'm just thinking out aloud here :wink: